server/src/main/java/com/linecorp/centraldogma/server/internal/replication/ZooKeeperCommandExecutor.java

```diff
@@ -718,12 +721,16 @@ private synchronized void replayLogs(long targetRevision) {
 
         long nextRevision = info.lastReplayedRevision + 1;
         for (;;) {
+            if (!canReplicate) {
```

Comment:
Question) Assuming a replica has been set with `canReplicate=false` will `canReplicate = true` be set for running servers without restarting? It seems like even if a `UpdateServerStatusCommand` is issued, `replayLogs` will never update this value due to the `break` statement for replicas